starlark-test/src/lib.rs

```diff
@@ -197,7 +200,7 @@ def assert_(cond, msg="assertion failed"):
                 io::stderr()
                     .write_all(
                         &format!(
-                            "Expected error '{}' at {}:{}, got success",
+                            "Expected error '{}' at {:?}:{}, got success",
```

Comment:
I believe this will quote the path in the error? It may be nicer to use `{}` and `path.display()` instead of `{:?}`, but I definitely don't feel strongly either way :)

Reply:
Ah yep, tbh I forgot path works like that.